source/hic_hal/stm32/stm32h743ii/sdk.c

```diff
@@ -20,103 +20,118 @@
  */
 
 #include "stm32h7xx.h"
-#include "DAP_config.h"
-#include "gpio.h"
-#include "daplink.h"
-#include "util.h"
 #include "cortex_m.h"
 
-//elee, pull in the version from cubemx main.c
-extern void SystemClock_Config(void);
-extern void MX_GPIO_Init(void);
-extern void MX_USB_OTG_FS_PCD_Init(void);
-extern void MX_USB_OTG_HS_PCD_Init(void);
-
-
 TIM_HandleTypeDef timer;
 uint32_t time_count;
 
 static uint32_t tim2_clk_div(uint32_t apb1clkdiv);
 
+static void Error_Handler(void)
+{
+    while (true) {}
+}
+
 /**
-    * @brief  Switch the PLL source from HSI to HSE bypass, and select the PLL as SYSCLK
-  *         source.
-  *         The system Clock is configured as follow :
-  *            System Clock source            = PLL (HSE bypass)
-  *            SYSCLK(Hz)                     = 72000000
-  *            HCLK(Hz)                       = 72000000
+  * @brief  The system Clock is configured as follow :
+  *            HSI Frequency(Hz)              = 64MHz
+  *            System Clock source            = PLL (HSI)
+  *            SYSCLK(Hz)                     = 240MHz
+  *            HCLK(Hz)                       = 120MHz
   *            AHB Prescaler                  = 1
-  *            APB1 Prescaler                 = 2
+  *            APB1 Prescaler                 = 1
   *            APB2 Prescaler                 = 1
-  *            HSE Frequency(Hz)              = 8000000
-  *            HSE PREDIV1                    = 1
-  *            PLLMUL                         = 9
-  *            Flash Latency(WS)              = 2
+  *            Flash Latency(WS)              = 1
```

Comment:
You increased bus speed (I think?), but (I'm guessing what this field means) decreased the flash wait states? Was it too high originally? Was this ever accurate?